packages/opentelemetry-tracing/src/Tracer.ts

```diff
@@ -50,7 +52,8 @@ export class Tracer implements api.Tracer {
   ) {
     const localConfig = mergeConfig(config);
```

Comment:
I believe `mergeConfig` also configures a default sampler, which means we are now configuring 2 if the user doesn't provide one.

Reply:
OH, thanks. Which one of the two would be authoritative? In any case I guess we should just have one.

Reply:
You should probably move your logic into mergeConfig. All the other defaults are there so it would be confusing to have the sampling default in another place

Reply:
good catch, thanks!